src/main/java/edu/ie3/simona/api/simulation/ExtSimulation.java

```diff
@@ -48,7 +48,13 @@ private boolean takeAndHandleMessage() throws InterruptedException {
 
     if (msg.getClass().equals(ActivityStartTrigger.class)) {
       final ActivityStartTrigger activityStartTrigger = (ActivityStartTrigger) msg;
-      List<Long> newTriggers = doActivity(activityStartTrigger.tick()); // this is blocking
+      List<Long> newTriggers;
+
+      if (activityStartTrigger.tick() == -1L) {
+        newTriggers = initialize();
+      } else {
+        newTriggers = doActivity(activityStartTrigger.tick()); // this is blocking
```

Comment:
Could you also adapt the comment here?
```suggestion
        newTriggers = doActivity(activityStartTrigger.tick()); // this is blocking until processing of this tick has finished
```